aws-distro-opentelemetry-node-autoinstrumentation/src/patches/aws/services/bedrock.ts

```diff
@@ -265,7 +284,7 @@ export class BedrockRuntimeServiceExtension implements ServiceExtension {
         if (requestBody.top_p !== undefined) {
           spanAttributes[AwsSpanProcessingUtil.GEN_AI_REQUEST_TOP_P] = requestBody.top_p;
         }
-      } else if (modelId.includes('mistral.mistral')) {
+      } else if (modelId.includes('mistral')) {
```

Comment:
We loosen this conditional because out of this [list](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-mistral-text-completion.html) for mistral models, one of the model ids starts with `mistral.mixtral` instead of `mistral.mistral`. The request/response body syntax is still the same for both.